bitnami/thanos/templates/alert-rule/compaction.yml

```diff
@@ -0,0 +1,120 @@
+{{- /*
+Generated from https://github.com/thanos-io/thanos/blob/main/examples/alerts/alerts.md
+*/ -}}
+{{- if  and .Values.metrics.enabled (or .Values.metrics.prometheusRule.default.create .Values.metrics.prometheusRule.default.compaction ) }}
```

Comment:
I explicitly declared all new values but I think it's not really clean because there are so many values. Is there any way I can make it cleaner?

Could you please elaborate `The files below also have a different name.`

Reply:
I think this is OK as-is. There are a lot of options, but they are all configurable by users, so they should be documented if we want to support them in this chart. Thanks!